torch/nn/functional.py

```diff
@@ -3293,7 +3294,7 @@ def grid_sample(input, grid, mode='bilinear', padding_mode='zeros', align_corner
         grid (Tensor): flow-field of shape :math:`(N, H_\text{out}, W_\text{out}, 2)` (4-D case)
                        or :math:`(N, D_\text{out}, H_\text{out}, W_\text{out}, 3)` (5-D case)
         mode (str): interpolation mode to calculate output values
-            ``'bilinear'`` | ``'nearest'``. Default: ``'bilinear'``
+            ``'bilinear'`` | ``'nearest'`` | ``'bicubic'``. Default: ``'bilinear'``
             Note: When ``mode='bilinear'`` and the input is 5-D, the interpolation mode
             used internally will actually be trilinear. However, when the input is 4-D,
             the interpolation mode will legitimately be bilinear.
```

Comment:
Add another note here for bicubic explaining that it's only available with 4D inputs, and briefly describing the approach (and choice of alpha).

Reply:
I mimic the [document](https://pytorch.org/docs/stable/nn.functional.html#upsample) in `upsample` to create a note block to briefly explain the method and also add some hints.